src/bazel/pkg_config_repository.bzl

```diff
@@ -86,6 +86,10 @@ def _exec_pkg_config(repo_ctx, flag):
         print("pkg-config is not found")  # buildifier: disable=print
         return []
     result = repo_ctx.execute([binary, flag] + repo_ctx.attr.packages)
+    # If result.stdout.strip() is empty, pkg-config will be re-executed with the --keep-system-cflags option added.
+    # Typically, -I/usr/include is returned, enabling Bazel to recognize packages that do not output cflags through pkg-config.
+    if result.stdout.strip() == "":
+        result = repo_ctx.execute([binary, flag] + ["--keep-system-cflags"] + repo_ctx.attr.packages)
```

Comment:
Can you wrap the comment so that each line is within 80 column?

While line length limit is [not strictly enforced in starlark files](https://bazel.build/rules/bzl-style#line-length), I personally feel it's convenient when reviewing code with side-by-side diff if each line is within 80 column.

Reply:
I see.
Wait a moments

Reply:
I wrapped comments and commit logs at 80 characters.